MAUI/Autocomplete/Selection.md

```diff
@@ -141,6 +141,56 @@ The selected items can be changed programmatically by using the [SelectedItems](
 
 {% endtabs %}
 
+### TokensWrapMode
+
+There are two ways to display multi selection items in the AutoComplete control. They are
+
+* Wrap
+* None
+
+#### Wrap mode
+
+When the TokensWrapMode is set to Wrap the selected items will be wrapped to the next line of the SfAutoComplete.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <editors:SfAutoComplete x:Name="autoComplete" 
+             HeightRequest="50"
+             WidthRequest="350"
+             ItemsSource="{Binding SocialMedias}"
+             SelectionMode="Multiple"
+             MaxDropDownHeight="250"
+             DisplayMemberPath="Name"
+             TextMemberPath="Name"
+             TokensWrapMode="Wrap" />
+
+{% endhighlight %}
+
+{% endtabs %}
+
+#### None Mode
+
+When the TokensWrapMode is set to None the selected item will be wrapped in horizontal orientation.
```

Comment:
When the TokensWrapMode is set to None, the selected item will be wrapped in a horizontal orientation.